generator/generator.yml

```diff
@@ -612,3 +612,42 @@ modules:
         lookup: lgpPduRcpEntrySysAssignLabel
       - source_indexes: [lgpPduAuxSensorIndex]
         lookup: lgpPduAuxSensorSysAssignLabel
+# Mikrotik Router
+#
+# http://download2.mikrotik.com/Mikrotik.mib
+  mikrotik:
+    walk:
+      - interfaces
+      - ifMIB
+      - laIndex
+      - sysUpTime
+      - sysDescr
+      - host
+      - mikrotik
+    lookups:
+      - source_indexes: [ifIndex]
+        lookup: ifAlias
+      - source_indexes: [ifIndex]
+        lookup: ifDescr
```

Comment:
Do you really need both of these? Usually for a given device one is enough.

Reply:
True, replaced by ifNames.